resources/conf/wdio-bstack-local-parallel-browsers.conf.js

```diff
@@ -0,0 +1,94 @@
+var defaults = require("./wdio.conf.js");
+var browserstack = require('browserstack-local');
+var _ = require("lodash");
+
+timeStamp = new Date().getTime();
+
+var overrides = {
+  user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
+  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACCESS_KEY',
+  specs: [
+    './src/test/suites/login/*.js',
+    './src/test/suites/offers/*.js',
+    './src/test/suites/product/*.js',
+    './src/test/suites/e2e/*.js',
+    './src/test/suites/user/*.js'
+  ],
+  host: 'hub.browserstack.com',
+  baseUrl: 'http://bs-local:3000/',
```

Comment:
this needs to be "http://bs-local.com:3000/".
For now lets change it to "http://localhost:3000/" and run on 2 browsers as there are some timeouts happening for iOS
```
[2-0] 2021-03-30T08:27:13.081Z WARN webdriver: Request timed out! Consider increasing the "connectionRetryTimeout" option.
[2-0] 2021-03-30T08:27:13.494Z WARN webdriver: Request failed with status 404 due to Session not started or terminated
[2-0] 2021-03-30T08:27:13.507Z WARN webdriver: Request failed with status 404 due to Session not started or terminated
[2-0] 2021-03-30T08:27:13.543Z ERROR webdriver: Request failed with status 404 due to Error: Session not started or terminated
```